app/src/main/java/org/techtown/soptseminar/FollowerAdapter.kt

```diff
@@ -0,0 +1,104 @@
+package org.techtown.soptseminar
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.content.Intent
+import android.service.autofill.UserData
+import android.view.LayoutInflater
+import android.view.MotionEvent
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import androidx.recyclerview.widget.RecyclerView
+import org.techtown.soptseminar.databinding.ItemFollowerSampleListBinding
+import java.util.*
+import kotlin.contracts.contract
+
+class FollowerAdapter() :
+    RecyclerView.Adapter<FollowerAdapter.FollowerViewHolder>(),
+    ItemTouchHelperCallback.OnItemMoveListener {
+
+    // 드래그 리스너 선언
+    private lateinit var dragListener: OnStartDragListener
+
+    // 클릭 리스너 선언
+    private lateinit var itemClickListener: ItemClickListener
+
+    val followerList =
+        mutableListOf<FollowerData>()
+
+    override fun getItemCount(): Int {
+        return followerList.size
+    }
```

Comment:
```suggestion
    override fun getItemCount(): Int = followerList.size
    
```
ㅎㅎ 사실 거기서 거기지만 이렇게하는것도 좋을거같습니다